cves/kernel/CVE-2021-3736.yml

```diff
@@ -173,8 +185,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: /
```

Comment:
Just a quick format thing - should this be `note: |` instead of the forward slash?